### PR TITLE
Akka Cluster Operator v.0.2.0

### PR DESCRIPTION
**Pull request by edwardcallahan**
PR'ing our Master for internal review prior to PR'ing upstream.
Still testing, last possible change is to hyphenate or not in:
```
      deployments:
        - name: akkaclusteroperator-deployment
```

Signed-off-by: Edward Callahan <ed.callahan@lightbend.com>

Thanks submitting your Operator. Please check below list before you create your Pull Request.
*************************************************
**Flat operator directory structure is obsolete from 23-rd of October 2019, only nested directory structure will be accepted.**
*************************************************

### New Submissions

* [X] Has you operator [nested directory structure](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#create-a-bundle)?
* [N/A] Have you selected the Project *Community Operator Submissions* in your PR on the right-hand menu bar? 
* [X] Are you familiar with our [contribution guidelines](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md)?
* [X] Have you [packaged and deployed](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md) your Operator for Operator Framework?
* [X] Have you tested your Operator with all Custom Resource Definitions?
* [X] Have you tested your Operator in all supported [installation modes](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#operator-metadata)?
* [X] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#sign-your-work)?

### Updates to existing Operators

* [X] Is your new CSV pointing to the previous version with the `replaces` property?
* [X] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#bundle-format) defined in the `package.yaml` ?
* [X] Have you tested an update to your Operator when deployed via OLM?
* [X] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#sign-your-work)?

### Your submission should not

* [X] Modify more than one operator
* [X] Modify an Operator you don't own
* [X] Rename an operator - please remove and add with a different name instead
* [X] Submit operators to both `upstream-community-operators` and `community-operators` at once
* [X] Modify any files outside the above mentioned folders
* [WIP] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [X] Description about the managed Application and where to find more information
2. [X] Features and capabilities of your Operator and how to use it
3. [X] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [X] Human readable name and 1-liner description about your Operator
* [X] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#categories)<sup>1</sup>
* [X] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [X] Links to the maintainer, source code and documentation
* [X] Example templates for all Custom Resource Definitions intended to be used
* [X] A quadratic logo

### CODE REVIEW

**Comment by edwardcallahan** [comment]
Different error from Travis than with v001.
```
resolvconf
git.checkout
1.70s$ git clone --depth=50 --branch=ACOv020 https://github.com/lightbend/community-operators.git lightbend/community-operators
0.01s
Setting environment variables from .travis.yml
$ export CHANGE_MINIKUBE_NONE_USER=true
$ export DISTRO_TYPE=upstream
0.01s$ source ~/virtualenv/python3.6/bin/activate
$ python --version
Python 3.6.7
$ pip --version
pip 19.0.3 from /home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/pip (python 3.6)
$ source scripts/ci/check-pr $DISTRO_TYPE
fatal: ambiguous argument '3647ddea939e...0bc78c6010f7': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
Invalid commit range.
```

File: community-operators/akka-cluster-operator/0.2.0/akka-cluster-operator.v0.2.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+    - name: Intro to Akka
+      url: https://doc.akka.io/docs/akka/current/guide/introduction.html
+    - name: Intro to Akka Cluster
+      url: https://doc.akka.io/docs/akka/current/common/cluster.html
+    - name: Akka Cluster demo application
+      url: https://github.com/lightbend/akka-java-cluster-openshift
+    - name: Deploying a Lagom application to OpenShift
+      url: https://developer.lightbend.com/guides/openshift-deployment/lagom/index.html
+  version: 0.2.0
+  replaces: akka-cluster-operator.v0.0.1
```
Comment (marcoderama) [inline community-operators/akka-cluster-operator/0.2.0/akka-cluster-operator.v0.2.0.clusterserviceversion.yaml]: Add newline?

**Review by marcoderama — APPROVED** [review]
Lgtm

**Comment by marcoderama** [comment]
I'd try a `push -f` to see if  that cures the build problem.  I've seen this happen before with at-different-times commits to a PR.  For some reason that has confused the build system.

**Pull request merged**